README.md

```diff
@@ -37,3 +38,6 @@ To ensure you have the correct dependencies run `npm i` in the command line
 
 ## Testing:
 To run e2e tests enter `codeceptjs run` in the command line
+
+## Service:
+see [spring boot template](https://github.com/hmcts/spring-boot-template) for more information.
```

Comment:
Please start sentence with capital S. Also do not link to spring-boot-template, link to `service/README.md`.